h/models/feature_cohort.py

```diff
@@ -22,16 +22,14 @@ def __init__(self, name):
         self.name = name
 
 
-FEATURECOHORT_USER_TABLE = sa.Table(
```

Comment:
So this is the only reference to this, the table name is used like `secondary="featurecohort_user"` but that should be fine, the actual postgres table name will stay the same :+1:.